plugins/processors/awsneuron/config.go

```diff
@@ -0,0 +1,19 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package awsneuron
+
+import (
+	"go.opentelemetry.io/collector/component"
+)
+
+type Config struct{}
+
+// Verify Config implements Processor interface.
+var _ component.Config = (*Config)(nil)
+
+// Validate does not check for unsupported dimension key-value pairs, because those
+// get silently dropped and ignored during translation.
+func (cfg *Config) Validate() error {
```

Comment:
How does it validate if it returns nil always?

Reply:
not relevant now, merged neuron processor with gpu